engine/src/test/java/com/redhat/cloud/notifications/DriftTestHelpers.java

```diff
@@ -52,8 +51,7 @@ public static EmailAggregation createEmailAggregation(String bundle, String appl
 
         emailActionMessage.setOrgId(DEFAULT_ORG_ID);
 
-        JsonObject payload = baseTransformer.toJsonObject(emailActionMessage);
-        aggregation.setPayload(payload);
+        aggregation.setPayload(TestHelpers.wrapActionToJsonObject(emailActionMessage));
```

Comment:
`baseTransformer` is no longer needed in this class.

Reply:
Removed!